internal/pkg/pluginengine/plugin.go

```diff
@@ -54,5 +57,10 @@ func Delete(tool *configmanager.Tool) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	renderInstanceIDtoOptions(tool)
 	return p.Delete(tool.Options)
 }
+
+func renderInstanceIDtoOptions(tool *configmanager.Tool) {
+	tool.Options["instanceID"] = interface{}(tool.InstanceID)
```

Comment:
redundant `interface{}`

Reply:
Yeah, the interface{}() can be deleted, but it's more readable to keep it maybe.